packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -6,6 +6,8 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:usage/src/uuid.dart'; // ignore: implementation_imports
```

Comment:
Maybe it would be better to duplicate this code instead of importing something private?  It looks pretty self-contained.